src/main/java/org/apache/commons/lang3/ObjectUtils.java

```diff
@@ -277,6 +277,33 @@ public static boolean isNotEmpty(final Object object) {
         return object != null ? object : defaultSupplier == null ? null : defaultSupplier.get();
     }
 
+    /**
+     * Checks if any value in the given array is {@code null}.
+     *
+     * <p>
+     * If all the values are {@code null} or the array is {@code null}
+     * or empty, then {@code true} is returned, otherwise {@code false} is returned.
```

Comment:
Wrong method doc. Shouldn't it be?
`If any of the values are {@code null} or the array is {@code null}, then {@code true} is returned, otherwise {@code false} is returned`